WordPress/Classes/Plugins/Views/PluginListItemView.swift

```diff
@@ -102,5 +144,21 @@ struct PluginListItemView: View {
         }
 
         static let noDescriptionAvailable: String = NSLocalizedString("site.plugins.list.item.noDescriptionAvailable", value: "The plugin author did not provide a description for this plugin.", comment: "The message displayed when a plugin has no description")
+
+        static let activate: String = NSLocalizedString("site.plugins.list.item.action.activate", value: "Activate", comment: "Button to activate a plugin")
```

Comment:
It should portably be something like `sitePluginsList.itemAction.deactivate`. The dots are to separate namespaces.